cognite/client/_api/assets.py

```diff
@@ -168,6 +170,7 @@ def list(
         root_external_ids: List[str] = None,
         asset_subtree_ids: List[int] = None,
         asset_subtree_external_ids: List[str] = None,
+        data_set_ids: List[Dict[str, Any]] = None,
```

Comment:
Need to add this to docstring below, preferably with example of what the Dict contains.

Reply:
Also needs to be added to filter